api/admin/coordinator.py

```diff
@@ -60,11 +61,54 @@ class CoordinatorLogEventsInline(
     show_change_link = True
 
 
+class CoordinatorForm(forms.ModelForm):  # type: ignore
+    class Meta:
+        model = models.Coordinator
+        fields = (
+            "is_validated",
+            "is_admin",
+            "additional_skills_comment",
+            "project_status",
+            "situational_status",
+            "status_since",
+        )
+
+
 class CoordinatorAdmin(VersionAdmin):
+    form = CoordinatorForm
     list_display = (
-        "__str__",
+        "get_personal_info_id",
+        "get_personal_info_full_name",
+        "get_is_validated",
+        "get_is_admin",
+        "get_additional_skills_comment",
+        "get_project_status",
+        "situational_status",
+        "get_status_since",
+        "get_communication_language_mode",
+        "mentor",
+        "get_comment",
         "active_groups_count",
     )
+
+    ordering = ["personal_info_id"]
+
+    list_filter = (
+        "is_validated",
+        "is_admin",
+        "project_status",
+        "situational_status",
+        "personal_info__communication_language_mode",
+        "mentor",
+    )
+
+    search_fields = (
+        "personal_info__id",
+        "personal_info__first_name",
+        "personal_info__last_name",
+        "additional_skills_comment",
+        "comment",
+    )
```

Comment:
I guess that here is where we should add something like `mentor__personal_info__id`, `mentor__personal_info__first_name`, `mentor__personal_info__last_name`

Reply:
done